src/components/Amount.tsx

```diff
@@ -52,6 +52,7 @@ interface Props {
   baseAssetValue?: string;
   fiatColor?: string;
   bold?: boolean;
+  style?: CSSProperties;
```

Comment:
We should avoid redeclaring react component props.
`const Amount: React.FC<Props> = ({}) => {}` is sufficient to type `style`

Reply:
That doesn't seem to be the case 😆

Reply:
My bad! `FC` doesn't add style attribute. Closing.